drivers/SmartThings/zigbee-switch/profiles/smartvill-switch-no-master-2.yml

```diff
@@ -0,0 +1,23 @@
+name: smartvill-switch-no-master-2
```

Comment:
The file names should not contain brand specific names. Please remove "smartvill" from the profile names. Please review our submission criteria https://developer.smartthings.com/docs/devices/hub-connected/certify-your-device#code-formatting-and-submission-criteria

Reply:
I dont believe these profiles are needed